### PR TITLE
feature:S3C-4200 Assume role backbeat API Integration - 2

### PR DESCRIPTION
**Pull request by saitharunthodupunuru**
Implemented `getCredentials` on `ScalityOsisService.java` to be used by all OSIS IAM flows. This method
* transforms `accountID` and prepares `AssumeRoleRequest`
* calls Vault with prepared `AssumeRoleRequest`

### CODE REVIEW

File: osis-core/src/test/java/com/scality/osis/utils/ScalityTestUtils.java [hunk 1]
```diff
@@ -20,6 +20,9 @@
                     "5b7e3259-aace-414c-bfd8-94daa0efefaf","6b7e3259-aace-414c-bfd8-94daa0efefaf",
                     "4b7e3259-aace-414c-bfd8-94daa0efefaf","3b7e3259-aace-414c-bfd8-94daa0efefaf")
     );
+    public static final String TEST_ROLE_ARN = "arn:aws:iam::12313265465789:role/osis";
```
Comment (anurag4DSB) [inline osis-core/src/test/java/com/scality/osis/utils/ScalityTestUtils.java:23]: Not a good account ID.
Account ID is 12 digits.

Reference: https://github.com/scality/Vault/blob/development/8.2/lib/IdGenerator.js#L17

**Review by anurag4DSB — CHANGES_REQUESTED** [review]
Just one minor change on account ID

File: osis-core/src/test/java/com/scality/osis/utils/ScalityTestUtils.java (hunk 1 above)
Comment (saitharunthodupunuru) [reply osis-core/src/test/java/com/scality/osis/utils/ScalityTestUtils.java]: done in 771e556f7555e54231774c097bd91a7c8688fd1a

**Review by anurag4DSB — APPROVED** [review]

**Review by ilkescality — APPROVED** [review]

**Pull request merged**